pennylane/devices/default_qubit_autograd.py

```diff
@@ -34,6 +37,9 @@ class DefaultQubitAutograd(DefaultQubitLegacy):
 
         pip install autograd
 
+    .. warning::
+        This device is deprecated. Use :class:`~.devices.DefaultQubit` instead; for example through ``qml.device("default.qubit")``.
```

Comment:
```suggestion
        This device is deprecated. Use :class:`~.devices.DefaultQubit` instead; for example through ``qml.device("default.qubit")``, which now supports backpropagation.
```
For the `default.qubit.torch` etc devices, the main reason to use them is for backprop, so I think it's nice to confirm to the reader here (and in the code warning) that we're not suggesting a regression, just a different way to access.